R/glmClassical.R

```diff
@@ -0,0 +1,1199 @@
+#
+# Copyright (C) 2013-2018 University of Amsterdam
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+#still to do: multinomial, ordinal, negative binomial, quasi
+
+GeneralizedLinearModel <- function(jaspResults, dataset = NULL, options, ...) {
+  if (options$family == "binomial") {
+    ready <- (options$dependent != "" && options$weights != "")
```

Comment:
```suggestion
    ready <- (options[["dependent"]] != "" && options[["weights"]] != "")
```
In general, we recommend using `[[` instead of `$` for subsetting the options list. `$` does partial matching which could lead to unexpected behavior

Reply:
will do!